holidays/countries/anguilla.py

```diff
@@ -0,0 +1,179 @@
+        - [Public Holidays For 2018](https://web.archive.org/web/20181127040220/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2019](https://web.archive.org/web/20191021020537/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2020](https://web.archive.org/web/20210205021056/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2021](https://web.archive.org/web/20210901183726/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2022](https://web.archive.org/web/20221011200016/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2024](https://web.archive.org/web/20241004074741/https://www.gov.ai/service/public-holidays-for-2024)
+        - [Public Holidays For 2025](https://web.archive.org/web/20250425025259/https://www.gov.ai/service/public-holidays-for-2025)
+        - [Public Holidays Regulations R.R.A. P130-1 as of DEC 15, 2000](https://web.archive.org/web/20250611054639/https://laws.gov.ai/storage/pdfs/2000%20AXA%20Revised%20Statutes%20and%20Regulations/PDF%20(Regulations)/P-R.R.A.s/P130-Public%20Holidays%20Regulations.pdf)
+        - [Public Holidays Regulations R.R.A. P130-1 as of DEC 15, 2010](https://web.archive.org/web/20250611055143/https://laws.gov.ai/storage/pdfs/2010%20AXA%20Revised%20Statutes%20and%20Regulations/PDF%20(Regulations)/P-R.R.A.s/P130-Public%20Holidays%20Regulations.pdf)
+    """
+
+    country = "AI"
+    default_language = "en_AI"
+    supported_languages = ("en_AI", "en_US")
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    # Declaration of independence May 30, 1967,
+    # but the 2000 revision is the earliest comprehensive legal update.
+    start_year = 2001
+    weekend = {SUN}
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, cls=AnguillaStaticHolidays)
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day(tr("New Year's Day")))
+
+        if self._year >= 2010:
+            # James Ronald Webster Day.
+            self._add_observed(self._add_holiday_mar_2(tr("James Ronald Webster Day")))
+
+        # Good Friday.
+        self._add_good_friday(tr("Good Friday"))
+
+        # Easter Sunday.
+        self._add_easter_sunday(tr("Easter Sunday"))
+
+        # Easter Monday.
+        self._add_easter_monday(tr("Easter Monday"))
+
+        # Labor Day.
+        self._add_observed(self._add_labor_day(tr("Labour Day")))
+
+        # Whit Monday.
+        whit_monday = self._add_whit_monday(tr("Whit Monday"))
+
+        name = (
+            # Queen's Birthday.
+            tr("Queen's Birthday")
+            if self._year <= 2022
+            # King's Birthday.
+            else tr("King's Birthday")
```

Comment:
As @KJhellico suggested above - `en_AI` kept the full name version while `en_US` uses the shortened ones
```suggestion
        name = (
            # Queen's Birthday.
            tr("Celebration of the Birthday of Her Majesty the Queen")
            if self._year <= 2022
            # King's Birthday.
            else tr("Celebration of the Birthday of His Majesty the King")
```